argo_workflows/README.md

```diff
@@ -0,0 +1,55 @@
+# Agent Check: Argo_Workflows
+
+## Overview
+
+This check monitors [Argo_Workflows][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.
+
+### Installation
+
+The Argo_Workflows check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Edit the `argo_workflows.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your argo_workflows performance data. See the [sample argo_workflows.d/conf.yaml][4] for all available configuration options.
```

Comment:
```suggestion
1. Edit the `argo_workflows.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your Argo_Workflows performance data. See the [sample argo_workflows.d/conf.yaml][4] for all available configuration options.
```

Reply:
Or just "Argo Workflows," to distinguish from the check itself?

Reply:
@jhgilbert I'm gonna submit a readme PR tmr where I'll address this.